openapi/openapi.yaml

```
@@ -105,6 +105,22 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/CapitalProjectPage"
+  /capital-projects/{managingCode}/{capitalProjectid}:
```

I missed this before but Id should be capitalized

```suggestion
  /capital-projects/{managingCode}/{capitalProjectid}:
```